spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -18,14 +18,17 @@
 import logging
 from typing import Sequence
 
-import torch
-
-from spinalcordtoolbox.reports import qc2
-from spinalcordtoolbox.deepseg import models, inference
 from spinalcordtoolbox.image import splitext, Image, check_image_kind
 from spinalcordtoolbox.utils.shell import SCTArgumentParser, Metavar, display_viewer_syntax, ActionCreateFolder
 from spinalcordtoolbox.utils.sys import init_sct, printv, set_loglevel, __version__, _git_info
 from spinalcordtoolbox.utils.fs import tmp_create
+from spinalcordtoolbox.utils.sys import lazy_import
+
+cuda = lazy_import('torch.cuda')
+
+qc2 = lazy_import('spinalcordtoolbox.reports.qc2')
```

Comment:
TODO: Replace this lazy import with `matplotlib` lazy imports inside both `qc2.py` and `qc.py`.